TFEngine.py

```diff
@@ -1344,48 +1344,80 @@ def _do_save(self):
         return False
     return True
 
-  def eval_model(self, output_file=None, output_per_seq_file=None):
+  def eval_model(self, output_file=None, output_per_seq_file=None, loss_name=None, output_per_seq_format=["seq_len", "score", "pos_score"]):
```

Comment:
Small other problem (and I'm quite sure PyCharm warns about this): You should not use mutable default values (such as a list). Make this a tuple instead.